frontend/glideinFrontendInterface.py

```diff
@@ -1191,9 +1191,16 @@ def createAdvertizeWorkFile(self, factory_pool, params_obj, key_obj=None, file_i
                     f"Factory requires {factory_trust}. ({request_cred.credential.path})"
                 )
                 continue  # Skip credentials that don't match the trust domain
-            if request_cred.req_idle == 0 and request_cred.req_max_run == 0:
-                logSupport.log.debug(f"Skipping credential with no work assigned. ({request_cred.credential.path})")
-                continue  # Skip credentials with no work assigned
+            # NOTE: This is a new behavior. If a credential has no work assinged to it or active glideins using it, no glideclient be avertized for it.
```

Comment:
# NOTE: Up to GWMS 3.10.x glideclient was always advertised. This is a new behavior.
# If a credential has no work assinged to it or active glideins using it, no glideclient be avertized for it.

Reply:
changed code